src/service/lootDrop.ts

```diff
@@ -204,8 +204,22 @@ export async function postLootDrop(
                           url: "attachment://opened.gif",
                       }
                     : undefined,
+                fields: [
+                    {
+                        name: "🎉 Geschenköffner",
```

Comment:
Das im Footer war einfach kopiert von dem Bot, woher das gesamte Geschenke-Ding geklaut war. `Geschenköffner` hört sich ungeil an, aber mir fällt auch nichts besseres ein. Vielleicht einfach nur ein "Geöffnet von"?

Reply:
Glücklicker Gewinner?

Reply:
Hab's in "Ehrenwerter Empfänger" umbenannt. Kann man ja immer noch ändern